allure-generator/webpack/webpack.prod.js

```diff
@@ -0,0 +1,86 @@
+const webpack = require("webpack");
+const webpackMerge = require("webpack-merge");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const WorkboxPlugin = require("workbox-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+const sass = require("sass");
+
+const commonConfig = require("./webpack.common.js");
+
+const utils = require("./utils.js");
+
+const postcssLoader = {
+  loader: "postcss-loader",
+  options: {
+    plugins: [require("autoprefixer")({ browsers: ["last 10 versions"] })],
```

Comment:
You don't need to specify `browsers` here, they should be picked up from the `package.json`